components/lib/biometric-prompt/src/main/java/mozilla/components/lib/auth/BiometricPromptFeature.kt

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.auth
+
+import android.content.Context
+import androidx.annotation.VisibleForTesting
+import androidx.biometric.BiometricManager.Authenticators.BIOMETRIC_WEAK
+import androidx.biometric.BiometricManager.Authenticators.DEVICE_CREDENTIAL
+import androidx.biometric.BiometricPrompt
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import mozilla.components.support.base.feature.LifecycleAwareFeature
+import mozilla.components.support.base.log.logger.Logger
+
+/**
+ * A [LifecycleAwareFeature] for the Android Biometric API to prompt for user authentication.
```

Comment:
```suggestion
 * A [LifecycleAwareFeature] for the Android Biometric API to prompt for user authentication. The prompt also requests support for the device PIN as a fallback authentication mechanism.
```

It might be useful to also describe that the prompt also supports the fallback PIN mechanism if the device does not support or has broken biometric hardware.